ext/redhat/logrotate

```diff
@@ -1,9 +1,23 @@
-/var/log/puppetlabs/puppet.log {
+# Generic logrotate config for Red Hat systems. Works with both official
+# Fedora and RHEL packages as well as PuppetLabs distributions.
+
+/var/log/puppetlabs/masterhttp.log /var/log/puppet/masterhttp.log {
+  compress
   missingok
   notifempty
-  create 0644 puppet puppet
+  # permissions/owner must be aligned with puppet/defaults.rb
+  create 0660 puppet puppet
+}
+
+/var/log/puppetlabs/puppetd.log /var/log/puppet/puppetd.log {
```

Comment:
Do we need the fallback log in the default logroate configuration? Documentation suggests otherwise:

      :desc => "The fallback log file. This is only used when the `--logdest` option
        is not specified AND Puppet is running on an operating system where both
        the POSIX syslog service and the Windows Event Log are unavailable. (Currently,
        no supported operating systems match that description.)